content/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services.md

```diff
@@ -76,90 +82,102 @@ In this guide we’re using the `eksctl` command to create an Amazon EKS cluster
 
-1. In <span style="white-space: nowrap; font-weight:bold;">service/loadbalancer-aws-elb.yaml</span>, add the following annotation to the existing or new `LoadBalancer` service:
+We also recommend enabling the PROXY Protocol for both the NGINX Plus Ingress Controller and your NLB target groups. This is used to forward client connection information. If you choose not to enable the PROXY protocol, see the [Appendix](#appendix).
 
-   ```yaml
-   service.beta.kubernetes.io/aws-load-balancer-type: nlb
-   ```
+### Configuring a `LoadBalancer` Service to Use NLB
 
-2. Run the following command:
+Apply the manifest <span style="white-space: nowrap; font-weight:bold;">deployments/service/loadbalancer-aws-elb.yaml</span> to create a `LoadBalancer` of type NLB:
 
    ```shell
-   kubectl apply -f service/loadbalancer-aws-elb.yaml
+   kubectl apply -f deployments/service/loadbalancer-aws-elb.yaml
    ```
 
 ### Enabling the PROXY Protocol
 
-1. Add the following keys to the <span style="white-space: nowrap; font-weight:bold;">common/nginx-config.yaml</span> config map file:
+1. Add the following keys to the <span style="white-space: nowrap; font-weight:bold;">deployments/common/nginx-config.yaml</span> config map file:
 
    ```yaml
    proxy-protocol: "True"
    real-ip-header: "proxy_protocol"
    set-real-ip-from: "0.0.0.0/0"
    ```
 
-2. Run the following command to update the config map:
+2. Run the following command to update the config map. This will enable the PROXY protocol for the NGINX Plus Ingress Controller:
 
    ```shell
-   kubectl apply -f common/nginx-config.yaml
+   kubectl apply -f deployments/common/nginx-config.yaml
    ```
 
-3. Enable the PROXY Protocol on the target group associated with the NLB created for your `LoadBalancer` service, by performing the steps in the **Enable Proxy Protocol** section of the [AWS documentation](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-target-groups.html#proxy-protocol).
+3. Enable the PROXY Protocol for the target groups linked to the NLB. Follow the steps in the **Enable proxy protocol** section of the [AWS documentation](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/edit-target-group-attributes.html#proxy-protocol) to do this.
 
 
 <span id="testing"></span>
 ## Testing
 
-1. Follow the [instructions](https://github.com/nginxinc/kubernetes-ingress/tree/master/examples/complete-example) for setting up our sample deployment of a demo app load balanced by the NGINX Plus Ingress controller.
-
-2. Navigate in a browser to **<http://cafe.example.com/coffee>** (or issue the `curl` command against that URL).
-
-3. Run the following command to display the name of the running Ingress pod:
+1. Get the DNS name of the NGINX Plus Ingress Controller with the command below. Find the value of the "LoadBalancer Ingress" in the output.
```

Comment:
```suggestion
1. Get the DNS name of NGINX Ingress Controller with the command below. Find the value of the "LoadBalancer Ingress" in the output.
```  

I correct this earlier in the file, but we don't prefix NGINX Ingress Controller with "the" if there's only one instance in the given topic. Within the context of referring to a specific instance, we would add "instance" as a suffix to the name, such as "the NGINX Ingress Controller instance".